localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -26,4 +26,7 @@
  principality_nightborne:0 "Principality"
 
  lord_nightborne_male:0 "Lord"
- lady_nightborne_female:0 "Lady" 
\ No newline at end of file
+ lady_nightborne_female:0 "Lady"
+
+ blood_troll_patriarch_male:0 "#"
```

Comment:
```suggestion
 blood_troll_patriarch_male:0 ""
```

Reply:
Are you sure about this? Leaving it empty adds an unsightly empty space before their name. With the `#` added, there's no space and it looks more pleasing to the eyes.

![image](https://user-images.githubusercontent.com/78575425/126687496-50220c98-1e5e-4c31-be9a-4a731d914442.png)